README.md

```diff
@@ -1,7 +1,31 @@
-# hackalog-backend
+# Hackalog-Backend
 
 ## Instructions for setting up this project
-- Create a `.env` file containing all the required keys
-- `pipenv shell`
-- `pipenv install`
-- Now you can apply the migrations and start the server
+* Fork and clone the project in your device.
```

Comment:
device can be changed to development environment

Reply:
I used most general terms to avoid confusion to anyone relatively newer to these terms 😄 .